src/styles/main.scss

```diff
@@ -1,3 +1,50 @@
-body {
-  margin: 0;
+@import './values';
+@import './mixins';
+@import './blocks/header';
+@import './blocks/icon';
+@import './blocks/menu';
+@import './blocks/nav';
+@import './blocks/page';
+@import './blocks/top-bar';
+@import './blocks/section-title';
+@import './blocks/product';
+@import './blocks/recommendet';
+@import './blocks/about-us';
+@import './blocks/categories';
+@import './blocks/contacts';
+
+.recommended {
+  margin-top: 64px;
+
+  @include onTablet {
+    margin-top: 120px;
+  }
+}
+
+.categories {
+  margin-top: 96px;
+
+  @include onTablet {
+    margin-top: 120px;
+  }
+
+  @include onDesktop {
+    margin-top: 128px;
+  }
+}
+
+.about-us {
+  margin-top: 64px;
+
+  @include onTablet {
+    margin-top: 120px;
+  }
+}
+
+.contacts {
+  margin-top: 104px;
+
+  @include onTablet {
+    margin-top: 120px;
+  }
```

Comment:
Why declare this here and not inside corresponding block definitions?

+ remove margins from BEM blocks